.github/workflows/on-main-branch-commit-push-liminal-node-to-ecr.yml

```diff
@@ -0,0 +1,80 @@
+---
+name: Builds and pushes aleph-node liminal to ECR on push to main branch
+
+on:
+  push:
+    branches:
+      - main
+      # temp for testing
+      - add-test-liminal-binary-to-workflows
+
+concurrency:
+  group: ${{ github.ref }}-${{ github.workflow }}
+  cancel-in-progress: true
+
+jobs:
+  build-liminal-node:
+    name: Build liminal node
+    uses: ./.github/workflows/_build-liminal-node.yml
+    secrets: inherit
+
+  push-liminal-node-image-to-ecr:
+    name: Push liminal node docker image to the ECR
+    needs: [build-liminal-node]
+    runs-on: ubuntu-20.04
```

Comment:
Are you not using `self-hosted` here on purpose?

Reply:
Yes, it's not needed, as it is in phase after merge and takes about 20 seconds anyway